docs/UserGuide.md

```diff
@@ -38,97 +38,204 @@
 
-> Deletes the person at the specified `INDEX`. 
+> Selects the task and modifies the task as done at the specified `INDEX`. 
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
+> Overwrites any of the specified fields ('LOCATION', 'DESCRIPTION'...) with the new values
+
+#### Mark a task as done : `mark`
+Marks the task as completed identified by the index number used in the last task listing. Completed task will be remove from the normal list and placed under archived list<br>
+Format: `mark INDEX [MORE_INDEX]`
+
+> Selects the task and marks the task as done at the specified `INDEX`. 
   The index refers to the index number shown in the most recent listing.<br>
   The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
 * `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+  `mark 2`<br>
+  Marks the 2nd task in the task book as completed.
+* `find CS2103` <br> 
+  `mark 1`<br>
+  Marks the 1st task in the results of the `find` command as completed.
 
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+#### Unmark a task as done : `unmark`
+Unmarks the task identified by the index number used in the last task listing.<br>
+Format: `Unmark INDEX [MORE_INDEX]`
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
+> Selects the task and marks the task as done at the specified `INDEX`. 
   The index refers to the index number shown in the most recent listing.<br>
   The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
 * `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+  `unmark 2`<br>
+  Unmarks the 2nd task in the task book as done.
+* `find CS2103` <br> 
+  `unmark 1`<br>
+  Unmarks the 1st task in the results of the `find` command as done.
+
+#### Undo the most recent operation : `undo`
+Undo the most recent command that was executed.<br>
+Format: `undo`  
+
+#### Redo the most recent undo operation : `redo`
+Redo the most recent command that was executed by the undo.<br>
+Format: `redo`  
+>Redo is unavailable if the most recent command is not undo
 
 #### Clearing all entries : `clear`
-Clears all entries from the address book.<br>
+Clears all entries from the Savvy Task.<br>
 Format: `clear`  
 
-#### Exiting the program : `exit`
-Exits the program.<br>
-Format: `exit`  
+#### Alias a keyword : `alias`
+Alias a keyword with shorter version of keyword <br>
```

Comment:
can you alias a command? e.g, alias `add` to `+` or something like that.
if so, please mention that. its a nice to have feature for advanced users.